alacritty/src/renderer/mod.rs

```diff
@@ -283,6 +291,75 @@ impl GlyphCache {
         FontDesc::new(desc.family.clone(), style)
     }
 
+    //pub fn font_metrics(&self) -> font::Metrics {
+    //    self.rasterizer
+    //        .metrics(self.font_key, self.font_size)
+    //        .expect("metrics load since font is loaded at glyph cache creation")
+    //}
+
```

Comment:
Hmm this comment is fairly useless as is.  Is this function used to benchmark the new code?

I think this should be either deleted or moved in a test file.